clients/banking/src/components/CardItemPhysicalDetails.tsx

```diff
@@ -1072,11 +1127,29 @@ export const CardItemPhysicalDetails = ({
                   },
                   () => (
                     <>
-                      <Space height={24} />
-
-                      <LakeButton color="current" onPress={() => setIsOrderModalOpen(true)}>
-                        {t("card.physical.order")}
-                      </LakeButton>
+                      {bindingUserError ? (
+                        <>
+                          <Space height={24} />
+
+                          <LakeTooltip content={t("card.tooltipConflict")} placement="bottom">
+                            <LakeButton
```

Comment:
I think pattern suggested by Zoon will be better here for maintenability. It will make update safe because it will not be possible to update button with tooltip without updating the one without.  
Or maybe we should update `LakeTooltip` with a new prop like `disable` so we could write:
```ts
<LakeTooltip content={t("card.tooltipConflict")} placement="bottom" disabled={!bindingUserError}>
  <LakeButton
    disabled={bindingUserError}
    color="current"
    onPress={() => setIsOrderModalOpen(true)}
  >
    {t("card.physical.order")}
  </LakeButton>
</LakeTooltip>
```
What do you think?